Source/Core/Common/TraversalServer.cpp

```diff
@@ -185,8 +185,9 @@ static const char* SenderName(sockaddr_in6* addr)
 static void TrySend(const void* buffer, size_t size, sockaddr_in6* addr)
 {
 #if DEBUG
-  printf("-> %d %llu %s\n", ((TraversalPacket*)buffer)->type,
-         (long long)((TraversalPacket*)buffer)->requestId, SenderName(addr));
+  const auto* packet = static_cast<const TraversalPacket*>(buffer);
+  printf("-> %d %llu %s\n", static_cast<int>(packet->type),
```

Comment:
I wonder if "replace `printf` with a `xxxx_LOG` statement" falls under "minor cleanup" - but that certainly shouldn't block this or anything. Especially when the Log Statement could get rid of the `#if DEBUG` (unless it means something significant for performance)